package-lock.json

```diff
@@ -31,7 +31,7 @@
     "ansi-styles": {
```

Comment:
should this file be `.gitignore`'d?

Reply:
Could we leave it here ? 

package-lock ensures that installing modules from the same package.json never results in two different installs for different users! 
The node community kinda encourages folks to not ignore it

Reply:
@owocki I made comments on your comments, feel free to leave more comments :)